cloud/passportauth/example/UserApi.ts

```diff
@@ -0,0 +1,37 @@
+import * as Promise from 'bluebird';
+import {BaseUser} from '../src/index';
+import UserDataRepository from './UserDataRepository';
+import {User} from './UserSeedData';
+
+/**
+ * A sample implementation of a base user api
```

Comment:
Is this really just an example? We plan on supplying supported implementations of interfaces, so maybe this would be it?